app/assets/v2/css/landing_page.css

```diff
@@ -840,6 +840,25 @@ body {
   max-width: 500px;
 }
 
+
+.lab-hero-bg {
+  position: relative;
+  align-content: center;
+  height: 200px;
+}
+
+.lab-hero-bg-container{
```

Comment:
This was already declared in labs.css

Reply:
Maybe is better to keep the lab styles in the lab.css file, I think this is not needed in the landing page.

Reply:
Removed